core/templates/components/common-layout-directives/common-elements/classroom-navigation-links.component.html

```diff
@@ -10,19 +10,17 @@
           <p>{{ 'I18N_CLASSROOM_NAVIGATION_LINKS_DESCRIPTION' | translate }}</p>
         </div>
         <div *ngFor="let classroom of classroomSummaries" class="classroom-nav-item-container">
-          <div class="classroom-item">
+          <div class="classroom-item" (click)="onClickClassroomCard(classroom.name)">
```

Comment:
Can you please rename this variable, to something describing its action?

Reply:
Done. I guess registerClassroomCardClickEvent is a better name for it.